src/server/schema.js

```diff
@@ -11,6 +11,7 @@ const esgqlTypeMapping = {
   byte: 'Int',
   double: 'Float',
   float: 'Float',
+  boolean: 'Boolean',
```

Comment:
how about just treat boolean as string for gql types?

either way, we will need to test before merging, especially to test use new Tube do ETL, and then see if Guppy/Portal can work (data explorer/study viewer)

One concern I have is that in data explorer we need to verify if we put a boolean field in filters, when selecting the filter, does it will send a string value back to guppy and causes problems. This can be testing in qa-covid19

Reply:
it is tested in qa-niaid @mfshao . We should use boolean if graphql support boolean.

Reply:
I don't want see it tested in one env for one feature and after merging it breaks another feature in another env. If we know it might potential be dangerous, we need to do more testing and verify things are ok, or fix more before merging. Don't rush to merge into master